### PR TITLE
Cli & find configure

### PR DESCRIPTION
**Pull request by RTSTALER**
Added support for cli, and search for configuration file

Commands:

Insert path
`
cli.ts console --path ./halva.js
`

The file will be found in the root of the project.
`
cli.ts console
`

### CODE REVIEW

**Comment by ByKeks** [comment]
When I do a simple `npm test` I get an error saying "TypeError [ERR_INVALID_ARG_TYPE]: The "id" argument must be of type string". Please fix it

File: src/cli.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+import yargs from 'yargs';
+import { run } from './Repl/repl'
+// tslint:disable-next-line: no-unused-expression
+yargs
+  .usage('Usage: $0 <cmd> [args]')
+  .command('console [path]', 'run console', (yargs) => {
+    yargs.positional('path', {
+      type: 'string',
+      // tslint:disable-next-line: object-literal-sort-keys
```
Comment (ByKeks) [inline src/cli.ts:10]: Don't use `disable-next-line` if it is not necessary.

File: src/cli.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      default: null,
+      describe: 'path to configure file'
+    })
+  // tslint:disable-next-line: only-arrow-functions
```
Comment (ByKeks) [inline src/cli.ts:14]: Don't use disable-next-line if it is not necessary.

File: package.json [hunk 3]
```diff
@@ -10,6 +10,8 @@
   "keywords": [],
   "scripts": {
     "console": "node --experimental-repl-await --require ts-node/register ./src/Repl/repl.ts",
+    "cli": "node --require ts-node/register ./src/cli.ts",
+    "cli-help": "node --require ts-node/register ./src/cli.ts --help",
```
Comment (ByKeks) [inline package.json:14]: Why reason use this? We can use `npm run cli -- --help` instead it

**Comment by RTSTALER** [comment]
> When I do a simple `npm test` I get an error saying "TypeError [ERR_INVALID_ARG_TYPE]: The "id" argument must be of type string". Please fix it

Resolve

**Pull request merged**